doc/getting-started.md

```diff
@@ -0,0 +1,390 @@
+> ```
+
+### Table of contents
+
+1. [Run a dogecoin node](#run-a-dogecoin-node)
+2. [Introduction to JSON-RPC API](#introduction-to-json-rpc-api)
+    * [Create a wallet](#create-a-wallet)
+    * [Verify balance](#verify-balance)
+    * [Send a transaction](#send-a-transaction)
+    * [Inspect blocks and transactions](#inspect-blocks-and-transactions)
+3. [Node configuration](#node-configuration)
+    * [Mainnet, testnet and regnet](#mainnet-testnet-and-regnet)
+    * [Data directory](#data-directory)
+    * [RPC credentials](#rpc-credentials)
+    * [Ports](#ports)
+    * [Memory](#memory)
+    * [Mode debug](#mode-debug)
+
+## Run a dogecoin node
+
+To start you node, you can run an headless server using `dogecoind` :
+```bash
+dogecoind -daemon
+```
+
+Or you can use a Grapich User Interface (GUI) with `dogecoin-qt` :
+```bash
+dogecoin-qt
+```
+
+Output will be recorded in `debug.log`, located in the [data directory](#data-directory).
+*Use `-help` to see all available options for each executables.*
+
+Your node should now be running, starting a *synchronization process* by downloading the entire blockchain from other nodes. This operation will take hours to complete... but your now part of the dogecoin network !
+
+> **Warning :** The rest of the guide will assume the use of an headless node. The RPC server is not exposed with `dogecoin-qt` until you activate the `-server` option.
+
+## Introduction to JSON-RPC API
+
+Dogecoin Core expose a JSON-RPC interface. With it, you can request information about blocks and transactions, send transactions to the networks, manage your wallet and so on...
+
+To interact with the interface, it relies on HTTP requests. Using urls like `http://user:password@127.0.0.1:22555/` and sending commands through json format is feasible.
+
+Some tools like libraries can be built on top of it. `dogecoin-cli` is the default shell client provided with Dogecoin Core to use this API.
+
+The RPC API documentation is available from [bitcoin](https://developer.bitcoin.org/reference/rpc/).  
+To have an overview of all commands, see also :
+
+```bash
+# List all commands
+dogecoin-cli help
+
+# Get help for a specific command
+dogecoin-cli help COMMAND
+```
+> **Tips** : Abuse of the help menus is allowed ! They can answer to multiple questions.
+
+### Create a wallet
+
+To receive or recover coins, you will need a wallet address and a private key. The *wallet address* is a shareable information to receive dogecoins from all over the world, while the *private key* is a sensitive information who give the ownership on coins associated to a public address.
+
+By default, you already have a wallet address and a private key available.
+
+List wallet addresses using `getaddressesbyaccount` :
+
+```bash
+dogecoin-cli getaddressesbyaccount ""
+[
+  "DBZc6qeBuo7g8La7L3Lhaf39MsJSgbzpMX"
+]
+```
+
+Use `getnewaddress` to generate a new wallet address :
+```bash
+dogecoin-cli getnewaddress
+DC13A8qUgAu5rTWweb44TwXZNQbxDYhyo2
```

Comment:
There is a problem about how to display commands, it's why I used the `$`, nothing to render properly a prompt with a smooth copy paste. I already played with different [syntax highlighting](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) [source [here](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml)] proposed by Github, sadly they do not seem to not offer something appropriate something appropriate for prompt + output convenient for copy paste.

Don't you think it's better to use `$` for readability over simplicity for copy paste ?  Or there is also console who may look correct:

with `console` keyword:
```console
shibe@doge:~$ echo "command" 43
[
  {
    "txid": "b869ed6606d52e6446dc12db02cf868ab693dd5b9f661116269536f0f8fa2433",
    "vout": 0,
    "address": "nnJDY1xCRgWQc7vBXHUPMPsEynuZW23Y3P",
  }
]
shibetoshi:~$ echo "command" 43
```
With `bash`:
```bash
$ echo "command" 43
[
  {
    "txid": "b869ed6606d52e6446dc12db02cf868ab693dd5b9f661116269536f0f8fa2433",
    "vout": 0,
    "address": "nnJDY1xCRgWQc7vBXHUPMPsEynuZW23Y3P",
  }
]
```

May answer others of your comments where you're suggesting to move code in separate blocks ? Not perfect for copy/pasting but good for readability

For the commented code, it could be :
```console
shibetoshi:~$ dogecoin-cli getnewaddress
DC13A8qUgAu5rTWweb44TwXZNQbxDYhyo2
```
Instead of:
```bash
dogecoin-cli getnewaddress
#return DC13A8qUgAu5rTWweb44TwXZNQbxDYhyo2
```

Reply:
Agree. The console markup will work well and makes it much clearer when there's a fully decorated prompt (much better than just `$`)

Reply:
Console actually highlight keyword in comments, look like it is not comments...
```console
# Get help for a specific command
shibetoshi:~$ dogecoin-cli help COMMAND

Reply:
It's in the same color when comment is fixed to `#`.
```console
#Get help for a specific command
shibetoshi:~$ dogecoin-cli help COMMAND
```